tests/nest_tests/test_multiple_continuous_input_ports.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# test_multiple_continuous_input_ports.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import os
+
+import nest
+import numpy as np
+
+from pynestml.frontend.pynestml_frontend import generate_nest_target
+
+
+class TestMultipleInputPorts:
+    """
+    Tests multiple continuous input ports in NEST.
+    """
+    def test_multiple_continuous_input_ports(self):
+        input_path = os.path.join(os.path.realpath(os.path.join(
+            os.path.dirname(__file__), "resources", "multiple_input_currents_neuron.nestml")))
+        generate_nest_target(input_path=input_path,
+                             target_path="target",
+                             logging_level="INFO",
+                             suffix="_nestml")
+        nest.ResetKernel()
+        nest.resolution = 0.1
+        nest.Install("nestmlmodule")
+        neuron = nest.Create("multiple_input_currents_neuron_nestml")
+        continuous_inputs = nest.GetStatus(neuron, "continuous_inputs")[0]
+
+        dc1 = nest.Create("dc_generator", params={"amplitude": 150.0})
+        nest.Connect(dc1, neuron, syn_spec={'receptor_type': continuous_inputs["I_1"]})
+
+        dc2 = nest.Create("dc_generator", params={"amplitude": 225.0})
+        nest.Connect(dc2, neuron, syn_spec={'receptor_type': continuous_inputs["I_2"]})
+
+        # multimeter{
+        mm = nest.Create("multimeter", params={"record_from": ["V_m1", "V_m2"]})
+        nest.Connect(mm, neuron)
+
+        nest.Simulate(10.)
+
+        v_m1 = nest.GetStatus(mm, "events")[0]["V_m1"]
+        v_m2 = nest.GetStatus(mm, "events")[0]["V_m2"]
+
+        np.testing.assert_allclose(v_m1[-1], 2370)
```

Comment:
Can these values be derived from the number of steps taken and DC amplitudes? By that token, why is this value 2370 and not 3000?

Reply:
The input is 150 and the script simulates for 10 ms at 0.1 ms resolution, so a total of 100 timesteps. There is a factor 1/5 in the script, so the total should be 150*100/3 = 3000, not 2370, right? Can you explain where the value 2370 comes from?

Reply:
Thanks for the updates!

Could you add this explanation as a comment in the code?

Could you also remove the ``print(v_m1, v_m2)`` call from the test script? Cheers!